dart/test/sentry_client_test.dart

```diff
@@ -122,6 +122,19 @@ void main() {
       expect(capturedEvent.stackTrace is SentryStackTrace, true);
     });
 
+    test('capture message defaults to info level', () async {
```

Comment:
Does it make sense to create a test like this for the static Sentry methods, too? To make sure that it gets passed down correctly. Or does a test like that already exist?

Reply:
`should capture message` test on SentryClient pass `SentryLevel.error` and asserts it, but yeah one coming from the Sentry static class makes sense, will do it, thanks